core/templates/pages/exploration-editor-page/exploration-editor-page.component.html

```diff
@@ -330,6 +338,7 @@ <h1>{{ getNavbarText() }}</h1>
 
   .navbar-icons {
     display: flex;
+    margin-bottom: 0;
```

Comment:
Is it doing a visual UI/style change? If yes, would you mind sharing the before/after screenshot?

Reply:
@DubeySandeep To fix the un-semantic code I had to change the html, that changed some stylings. So to fix that UI change I have added the style to keep it the same. Here is the screenshot of the same UI:
![Screenshot from 2023-10-31 11-34-42](https://github.com/oppia/oppia/assets/96219910/038b9048-ddd1-46f4-8325-99ad857e9398)